components/TaskCard/TaskCard.tsx

```diff
@@ -28,7 +29,10 @@ import { Countdown } from 'components/Countdown';
 import styles from './TaskCard.module.scss';
 
 export interface TaskCardProps {
-  proposal: CouncilProposalMeta | DemocracyProposalMeta;
+  proposal:
+    | CouncilProposalMeta
+    | DemocracyProposalMeta
+    | EthGovernanceProposalMeta;
   currentBlock: number | null;
```

Comment:
nit

maybe here we can get currentBlock from atom instead of passing it down as props

Reply:
I have no idea actually). It isn't a part of the PR. 🙃